web/source/keymanweb.js

```diff
@@ -2904,14 +2904,14 @@
     var n,Ln,lgCode;
```

Comment:
On the two uses:

- 2921:  This is used to force selection of a keyboard on mobile devices if none is otherwise set.  I'm guessing KMEA/KMEI have their default keyboard actually in the keyboard list?

- 2925:  This sets the selected keyboard in a cookie so that it remembers the languages if we refresh the page.  Not so helpful when testing, but pretty helpful for actual usage.

I'm confident that both are safe to refactor to PLgCode.